lib/twingly/search/parser.rb

```diff
@@ -3,6 +3,11 @@
 module Twingly
   module Search
     class Parser
+      # Parse an API response body.
+      #
+      # @param [String] document containing an API response XML.
+      # @raise [Error] which error depends on the API response (see {Error.from_api_response_message}).
+      # @return [Result] containing the result for this.
```

Comment:
> What's "this" in this case?

I don't remember, feels like something is missing at the end of the sentence. I'll just remove it :)